scripts/pre_push_hook.py

```diff
@@ -124,10 +124,17 @@ def _get_remote_name():
         raise ValueError(err)
 
     if not remote_num:
-        print ('Warning: Please set upstream for the lint checks to run '
-               'efficiently. You can learn more about it here -> '
-               'https://git-scm.com/book/en/v2/Git-Branching-Remote-Branches\n')
-        return
+        raise Exception(
+            'Error: Please set upstream for the lint checks to run '
+            'efficiently. To do that follow these steps:\n'
+            '1. Run the command \'git remote\'\n'
```

Comment:
Perhaps `git remote -v` since that gives more visibility into the current remotes and will be useful debug info in the future if anything goes wrong.

Reply:
Done.